src/routes/index.ts

```diff
@@ -0,0 +1,7 @@
+import express , {Request, Response, Router} from 'express';
```

Comment:
If you want to follow the architecture, the recommendation is to use a file named `src/app/app.router.ts`. 

```
+ entity-one
+ entity-two
- app.database.ts
- app.router.ts
- app.server.ts
```

Reply:
Done in 36af5b3